src/Microsoft.Health.Fhir.Anonymizer.Shared.CommandLineTool/FilesAnonymizerForJsonFormatResource.cs

```diff
@@ -62,14 +62,16 @@ public async Task AnonymizeAsync()
             stopWatch.Start();
 
             int completedCount = 0;
+            int skippedCount = 0;
             int failedCount = 0;
             Progress<BatchAnonymizeProgressDetail> progress = new Progress<BatchAnonymizeProgressDetail>();
             progress.ProgressChanged += (obj, args) =>
```

Comment:
i feel ProgressChanged is an event should be defined in FhirPartitionedExecutor

Reply:
We can refine it in another PR.